dozer-sql/src/pipeline/mod.rs

```diff
@@ -13,3 +13,16 @@ mod window;
 
 #[cfg(test)]
 mod tests;
+
+#[cfg(feature = "onnx")]
+pub mod udfs_errors;
+
+#[derive(Clone, Debug)]
+pub struct DozerSession(pub std::sync::Arc<ort::Session>);
```

Comment:
Does this compile without onnx feature?

Reply:
will move to the onnx like comment below